os/arch/xtensa/include/esp32/rtc_io_struct.h

```diff
@@ -0,0 +1,308 @@
+		} status;
+		union {
+			struct {
+				uint32_t reserved0 :14;
+				uint32_t w1ts :18;	/*GPIO0~17 interrupt status write 1 to set */
+			};
+			uint32_t val;
+		} status_w1ts;
+		union {
+			struct {
+				uint32_t reserved0 :14;
+				uint32_t w1tc :18;	/*GPIO0~17 interrupt status write 1 to clear */
+			};
+			uint32_t val;
+		} status_w1tc;
+		union {
+			struct {
+				uint32_t reserved0 :14;
+				uint32_t in :18;	/*GPIO0~17 input value */
+			};
+			uint32_t val;
+		} in_val;
+		union {
+			struct {
+				uint32_t reserved0 :2;
+				uint32_t pad_driver :1;	/*if set to 0 : normal output  if set to 1 : open drain */
+				uint32_t reserved3 :4;
+				uint32_t int_type :3;	/*if set to 0 : GPIO interrupt disable  if set to 1 : rising edge trigger  if set to 2 : falling edge trigger  if set to 3 : any edge trigger  if set to 4 : low level trigger  if set to 5 : high level trigger */
+				uint32_t wakeup_enable :1;	/*GPIO wake up enable  only available in light sleep */
+				uint32_t reserved11 :21;
+			};
+			uint32_t val;
+		} pin[18];
+		union {
+			struct {
+				uint32_t sel0 :5;
+				uint32_t sel1 :5;
+				uint32_t sel2 :5;
+				uint32_t sel3 :5;
+				uint32_t sel4 :5;
+				uint32_t no_gating_12m :1;
+				uint32_t reserved26 :6;
+			};
+			uint32_t val;
+		} debug_sel;
+		uint32_t dig_pad_hold;	/*select the digital pad hold value. */
+		union {
+			struct {
+				uint32_t reserved0 :30;
+				uint32_t hall_phase :1;	/*Reverse phase of hall sensor */
+				uint32_t xpd_hall :1;	/*Power on hall sensor and connect to VP and VN */
+			};
+			uint32_t val;
+		} hall_sens;
+		union {
+			struct {
+				uint32_t reserved0 :4;
+				uint32_t sense4_fun_ie :1;	/*the input enable of the pad */
+				uint32_t sense4_slp_ie :1;	/*the input enable of the pad in sleep status */
+				uint32_t sense4_slp_sel :1;	/*the sleep status selection signal of the pad */
+				uint32_t sense4_fun_sel :2;	/*the functional selection signal of the pad */
+				uint32_t sense3_fun_ie :1;	/*the input enable of the pad */
+				uint32_t sense3_slp_ie :1;	/*the input enable of the pad in sleep status */
+				uint32_t sense3_slp_sel :1;	/*the sleep status selection signal of the pad */
+				uint32_t sense3_fun_sel :2;	/*the functional selection signal of the pad */
+				uint32_t sense2_fun_ie :1;	/*the input enable of the pad */
+				uint32_t sense2_slp_ie :1;	/*the input enable of the pad in sleep status */
+				uint32_t sense2_slp_sel :1;	/*the sleep status selection signal of the pad */
+				uint32_t sense2_fun_sel :2;	/*the functional selection signal of the pad */
+				uint32_t sense1_fun_ie :1;	/*the input enable of the pad */
+				uint32_t sense1_slp_ie :1;	/*the input enable of the pad in sleep status */
+				uint32_t sense1_slp_sel :1;	/*the sleep status selection signal of the pad */
+				uint32_t sense1_fun_sel :2;	/*the functional selection signal of the pad */
+				uint32_t sense4_mux_sel :1;	/*�1� select the digital function  �0�slection the rtc function */
```

Comment:
check encoding format and content in the comments

Reply:
fixed, thanks.